modin/core/storage_formats/pandas/parsers.py

```diff
@@ -642,7 +642,7 @@ def parse(fname, **kwargs):
         ]
 
 
-class ParquetFileToRead(NamedTuple):
+class ParquetFileToRead:
```

Comment:
why make this a class?

Reply:
Currently, unidist on MPI doesn't handle serialization/deserialization of `NamedTuple` correctly so I had to make it a class.

Reply:
@YarShev have you fixed this issue?

Reply:
Are you going to create an issue for this?

Reply:
please file an issue so we won't forget

Reply:
https://github.com/modin-project/modin/issues/5273

Reply:
https://github.com/modin-project/modin/issues/5273